tools/wdio/plugins/testing-server/test-handle-connector.mjs

```diff
@@ -50,6 +50,36 @@ export class TestHandleConnector {
     }
   }
 
+  /**
+   * Schedules a reply to a server request
+   * @param {'assetServer'|'bamServer'} serverId Id of the server the request will be received on
+   * @param {ScheduledReply} scheduledReply The reply options to apply to the server request
+   */
+  async scheduleReply (serverId, scheduledReply) {
+    await fetch(`${this.#commandServerBase}/test-handle/${this.#testId}/scheduleReply`, {
+      method: 'POST',
+      body: JSON.stringify({
+        serverId,
+        scheduledReply: { ...scheduledReply, test: SerAny.serialize(scheduledReply.test) }
+      }),
+      headers: { 'content-type': 'application/json' }
+    })
+  }
+
+  /**
+   * Clears all scheduled replies for the given server.
+   * @param {'assetServer'|'bamServer'} serverId Id of the server the request will be received on
+   */
+  async clearScheduledReplies (serverId) {
+    await fetch(`${this.#commandServerBase}/test-handle/${this.#testId}/clearScheduledReplies`, {
+      method: 'POST',
+      body: JSON.stringify({
+        serverId
+      }),
+      headers: { 'content-type': 'application/json' }
+    })
+  }
+
```

Comment:
Per discussion is this being used?

Reply:
I am adding a flag to allow setting a reply as permanent in my PR so this might come in handy at some point. Doesn't hurt anything to leave it.